acceptance-tests/build.gradle

```diff
@@ -48,13 +53,33 @@ dependencies {
 
 test.enabled = false
 
-task acceptanceTest(type: Test) {
+task downloadHashicorpVault(type: Download) {
+  src {
+    // see gradle.properties for Hashicorp Vault URL
+    return project.property("hashicorpVaultUrl.${osdetector.os}")
+  }
+  dest new File(buildDir, "hashicorp.zip")
+  onlyIfModified true
+}
+
+task downloadAndExtractHashicorpVault(dependsOn: downloadHashicorpVault, type: Copy) {
```

Comment:
nit: is this downlaodAnd - or is this just the extract - the dependency is documented so need to factor into name (IMHO)

Reply:
it is extract only and depends on the download task. Updating the names to reflect it better.